### PR TITLE
fixes to spec as defined in BGDIINF_SB-1405_spec_cleanup

### PR DESCRIPTION
**Pull request by boecklic**
pls just make review comments and approve the PR, otherwise I will not be able to merge it in the evening without requiring a re-review. Thnx

### CODE REVIEW

**Comment by boecklic** [comment]
> _No description provided._

pls just make review comments and approve the PR, otherwise I will not be able to merge it in the evening without requiring a re-review. Thnx

File: spec/static/openapi.yaml [hunk 1]
```diff
@@ -512,7 +518,16 @@ components:
         description:
           $ref: "#/components/schemas/description"
           description: A description of the features in the collection
-          example: An address.
+          example: |-
```
Comment (ltshb) [inline spec/static/openapi.yaml:521]: Just code style, I would use `>-` instead of `|-`

File: spec/static/openapitransactional.yaml [hunk 2]
```diff
@@ -580,7 +586,16 @@ components:
         description:
           $ref: "#/components/schemas/description"
           description: A description of the features in the collection
-          example: An address.
+          example: |-
```
Comment (ltshb) [inline spec/static/openapitransactional.yaml:589]: Use `>-`

**Review by ltshb — APPROVED** [review]

File: spec/static/openapi.yaml (hunk 1 above)
Comment (boecklic) [reply spec/static/openapi.yaml:521]: Hmm, VS Code shows me an error if I put `|>` any idea why?

**Pull request merged**